docs/modules/migrate/pages/dmt-view-data.adoc

```diff
@@ -1,20 +1,22 @@
 = Verify the Migrated Data
-:description: To ensure that the migration includes all data, you can verify the size of the map in the target cluster.
+:description: To ensure that the migration includes all data, verify the size of the map or replicated map in the target cluster and check that any specified CP data structure exists.
 
 {description}
 
-You can verify the size of the map in the target cluster as follows:
+You can verify the size of the map, replicated map and check whether the specified CP data structures exist in the target cluster as follows:
 
 * Use the Hazelcast Management Center
 +
+NOTE: You cannot use Hazelcast CLC to check whether CP data structures exist. To do this, you must use Hazelcast Management Center.
++
 To use the Hazelcast Management Center, you can use either of the following methods:
 +
 ** Check the target map size, as described in the https://docs.hazelcast.com/management-center/5.3/data-structures/map[Maps^] section of the Hazelcast Management Center documentation
 ** Check the map entries, as described in the https://docs.hazelcast.com/management-center/5.3/data-structures/map#map-browser[Exploring Map Entries^] section of the Hazelcast Management Center documentation
 
 * Use Hazelcast CLC 
 +
-To use Hazelcast CLC to verify the migrated map size, enter the following command in your terminal:
+To use Hazelcast CLC to verify the migrated map or replicated map size, enter the following command in your terminal:
```

Comment:
not sure if this is correct. "--map" should be for the imaps. let me confirm it with Yuce

Reply:
confirmed that clc does not support replicated map

Reply:
[e3b6c3b](https://github.com/hazelcast/hz-docs/pull/936/commits/e3b6c3b17d71b48118a3cf9c9e2e607c215ed112)